hles/extraction/src/main/scala/org/broadinstitute/monster/dap/ExtractionPipelineBuilder.scala

```diff
@@ -104,12 +104,12 @@ class ExtractionPipelineBuilder(
         _.applyKvTransform(ParDo.of(lookupFn)).flatMap(kv => kv.getValue.fold(throw _, _.arr))
       }
 
-    StorageIO.writeJsonLists(
+    StorageIO.writeJsonListsGeneric(
       extractedRecords,
       "Write records",
       s"${args.outputPrefix}/${subDir}/records"
     )
-    StorageIO.writeJsonLists(
+    StorageIO.writeJsonListsGeneric(
```

Comment:
What's the distinction here?

Reply:
This is a new method @raaidbroad introduced in the latest version of ingest-utils. Briefly, writeJsonLists is generic in `[M : Encoder]` while `writeJsonListsGeneric` is typed to `SCollection[Msg]`. I think this is effectively the same codepath, just a new name.